tests/tools/context.py

```diff
@@ -2,9 +2,11 @@
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
 from dpgen.tools.run_report import *
-
+machine_file = 'machine_fp_single.json'
 def my_file_cmp(test, f0, f1):
     with open(f0) as fp0 :
         with open(f1) as fp1:
             test.assertTrue(fp0.read() == fp1.read())
 
+def setUpModule():
+    os.chdir(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
The name of the function is not corresponding to what it is doing

Reply:
This is copied from https://github.com/deepmodeling/dpgen/blob/da494650314b3eebcf31ff20a7c195b309cb1a6a/tests/generator/context.py#L32

Reply:
Do you think we should modify them together.

Reply:
Agree with you. I will fix other `setUpModule` in another PR.

Reply:
The name is reserved for UTs and helps to change dir to the file dir. So we should not change it. Please see 
https://docs.python.org/3/library/unittest.html#setupmodule-and-teardownmodule